changeset:   158:2db1dbb75557
user:        John Donoghue
date:        Thu Jan 09 16:38:52 2020 -0500
summary:     * inst/base64encode.m: dont use depreciated toascii

--- a/inst/base64encode.m
+++ b/inst/base64encode.m
@@ -46,7 +46,7 @@
   endif
   
   if (ischar (X))
-    X = toascii (X);
+    X = double (X);
   endif
   
   if (any (X != fix (X)) || any (X < 0 | X > 255))

